pandera/pandera.py

```diff
@@ -615,29 +741,40 @@ def __repr__(self):
 
 def check_input(schema, obj_getter=None):
     """Validate function argument when function is called.
-
+    
     This is a decorator function that validates the schema of a dataframe
     argument in a function. Note that if a transformer is specified by the
     schema, the decorator will return the transformed dataframe, which will be
     passed into the decorated function.
 
-    Parameters
-    ----------
-    schema : DataFrameSchema|SeriesSchema
-        dataframe/series schema object
-    obj_getter : int|str|None
-        if int, obj_getter refers to the the index of the pandas
-        dataframe/series to be validated in the args part of the function
-        signature. If str, obj_getter refers to the argument name of the pandas
-        dataframe/series in the function signature. This works even if the
-        series/dataframe is passed in as a positional argument when the
-        function is called. If None, assumes that thedataframe/series is the
-        first argument of the decorated function
-
+    :param schema: dataframe/series schema object
+    :type schema: DataFrameSchema|SeriesSchema
+    :param obj_getter:  (Default value = None) if int, obj_getter refers to the
+        the index of the pandas dataframe/series to be validated in the args
+        part of the function signature. If str, obj_getter refers to the
+        argument name of the pandas dataframe/series in the function signature.
+        This works even if the series/dataframe is passed in as a positional
+        argument when the function is called. If None, assumes that the
+        dataframe/series is the first argument of the decorated function
+    :type obj_getter: int|str|None
+    
     """
 
+    # noinspection PyUnusedLocal
     @wrapt.decorator
     def _wrapper(fn, instance, args, kwargs):
+        """This function wrapper called each time a decorated function is
```

Comment:
I don't think private functions need docstrings, esp. if it conforms to an idiomatic design pattern